lib/std/io.zig

```diff
@@ -46,14 +46,20 @@ fn getStdOutHandle() os.fd_t {
     return os.STDOUT_FILENO;
 }
 
+pub const StdOutType = if (builtin.os.tag == .windows) ConsoleWriter(4096) else File;
```

Comment:
I really don't like introducing these types for all platforms just because of Windows. I think an internal solution that works without introducing any new public APIs would be much better even if it may mean that on Windows some structs need to have more fields or checks.